soh/assets/soh_assets.h

```diff
@@ -162,3 +162,22 @@ static const ALIGN_ASSET(2) char gFileSelLanguageGERTex[] = dgFileSelLanguageGER
 
 #define dgEmptyTexture "__OTR__textures/virtual/gEmptyTexture"
 static const ALIGN_ASSET(2) char gEmptyTexture[] = dgEmptyTexture;
+
+// Custom Tunic Models
+#define dgLinkChildKokiriTunicSkel "__OTR__objects/object_link_child_kokiri/gLinkChildKokiriTunicSkel"
+static const ALIGN_ASSET(2) char gLinkChildKokiriTunicSkel[] = dgLinkChildKokiriTunicSkel;
+
+#define dgLinkChildGoronTunicSkel "__OTR__objects/object_link_child_goron/gLinkChildGoronTunicSkel"
+static const ALIGN_ASSET(2) char gLinkChildGoronTunicSkel[] = dgLinkChildGoronTunicSkel;
+
+#define dgLinkChildZoraTunicSkel "__OTR__objects/object_link_child_zora/gLinkChildZoraTunicSkel"
+static const ALIGN_ASSET(2) char gLinkChildZoraTunicSkel[] = dgLinkChildZoraTunicSkel;
+
+#define dgLinkAdultKokiriTunicSkel "__OTR__objects/object_link_boy_kokiri/gLinkAdultKokiriTunicSkel"
+static const ALIGN_ASSET(2) char gLinkAdultKokiriTunicSkel[] = dgLinkAdultKokiriTunicSkel;
+
+#define dgLinkAdultGoronTunicSkel "__OTR__objects/object_link_boy_goron/gLinkAdultGoronTunicSkel"
+static const ALIGN_ASSET(2) char gLinkAdultGoronTunicSkel[] = dgLinkAdultGoronTunicSkel;
+
+#define dgLinkAdultZoraTunicSkel "__OTR__objects/object_link_boy_zora/gLinkAdultZoraTunicSkel"
+static const ALIGN_ASSET(2) char gLinkAdultZoraTunicSkel[] = dgLinkAdultZoraTunicSkel;
```

Comment:
I think it'd be better to use the vanilla object name and only have the skeleton name itself be different.
```suggestion
// Custom Tunic Models
#define dgLinkChildKokiriTunicSkel "__OTR__objects/object_link_child/gLinkChildKokiriTunicSkel"
static const ALIGN_ASSET(2) char gLinkChildKokiriTunicSkel[] = dgLinkChildKokiriTunicSkel;

#define dgLinkChildGoronTunicSkel "__OTR__objects/object_link_child/gLinkChildGoronTunicSkel"
static const ALIGN_ASSET(2) char gLinkChildGoronTunicSkel[] = dgLinkChildGoronTunicSkel;

#define dgLinkChildZoraTunicSkel "__OTR__objects/object_link_child/gLinkChildZoraTunicSkel"
static const ALIGN_ASSET(2) char gLinkChildZoraTunicSkel[] = dgLinkChildZoraTunicSkel;

#define dgLinkAdultKokiriTunicSkel "__OTR__objects/object_link_boy/gLinkAdultKokiriTunicSkel"
static const ALIGN_ASSET(2) char gLinkAdultKokiriTunicSkel[] = dgLinkAdultKokiriTunicSkel;

#define dgLinkAdultGoronTunicSkel "__OTR__objects/object_link_boy/gLinkAdultGoronTunicSkel"
static const ALIGN_ASSET(2) char gLinkAdultGoronTunicSkel[] = dgLinkAdultGoronTunicSkel;

#define dgLinkAdultZoraTunicSkel "__OTR__objects/object_link_boy/gLinkAdultZoraTunicSkel"
static const ALIGN_ASSET(2) char gLinkAdultZoraTunicSkel[] = dgLinkAdultZoraTunicSkel;
```

Reply:
so i think i recall something about this being preferably different so that you can have the skeleton reference new sections in the otr without just straight up overriding any normal skeleton but open to changing it depending on how much of a difference it makes (since this just indiscriminately overrides the skeleton if its links)

Reply:
Yeah, I forgot that fast64 doesn't add the skeleton name on every additional file so they can override eachother, unfortunate but for now we can use the different object names.